### ISSUE TITLE
using @starbeam/react Component breaks vite css module auto reload

### ISSUE THREAD

**Issue by patricklx** [starbeamjs/starbeam#118]
i a react vite project
if I have a

```jsx
import styles from './App.module.css';

...
return Component(() => (<div className={styles.app}> </div>));
```
auto reload of css will not work anymore...

however, if I use useReactive it works

**Comment by NullVoxPopuli** [comment]
hello! does the latest `unstable` publish have the same issue?

**Comment by patricklx** [comment]
cannot install
```
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@domtree/flavors@workspace:^" is in the dependencies but no package named "@domtree/flavors" is present in the workspace

This error happened while installing the dependencies of @starbeam/react@0.8.10-unstable.35f0b99
 at @starbeam/debug@0.8.10-unstable.35f0b99
```

**Comment by patricklx** [comment]
now failing because of https://github.com/vitejs/vite/issues/14169
this is inside the vitests of @starbeam/debug...
is there not way to strip out the test parts when published?

**Comment by NullVoxPopuli** [comment]
There will be. We currently don't have a production build were happy with, but that'll happen very soon.

**Comment by NullVoxPopuli** [comment]
what's your build environment like?

if you using esm.sh, like here: https://jsbin.com/kivuqal/1/edit?html,output it can compile away the import.meta stuff.

otherwise, you may need something like https://iendeavor.github.io/import-meta-env/guide/getting-started/introduction.html

**Comment by NullVoxPopuli** [comment]
yeah, there's been a streamlining of concepts since the docs were written -- here is a current example: https://github.com/starbeamjs/starbeam/blob/main/demos/react/src/components/formatter/DateFormatter-v1.tsx#L15
